src/components/Footer/FooterLinks/index.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import boleto from './icons/boleto.svg';
+import elo from './icons/elo.svg';
+import mastercard from './icons/mastercard.svg';
+import paypal from './icons/paypal.svg';
+import visa from './icons/visa.svg';
+import telephone from './icons/telephone.svg';
+import email from './icons/email.svg';
+import {
+    FooterContainer,
+    FooterCategories,
+    CategoryLinks,
+    SupportInfo,
+    FooterImages,
+} from './styles';
+
+const FooterLinks = [
```

Comment:
We only use variables starting with an uppercase letter to components (Something that will be rendered like this `<FooterImages />`. It should be `footerLinks`